app/util/vehicleRentalUtils.js

```diff
@@ -47,7 +47,7 @@ export const getRentalNetworkId = networks => {
 };
 
 export const getRentalNetworkConfig = (networkId, config) => {
-  if (!networkId || !networkId.toLowerCase) {
+  if (!networkId || !networkId.toLowerCase || !config) {
```

Comment:
can config really be null? If it is should the code crash?

This function is quite poor in general. It makes lots of nonsense checks and tries to work no matter what. Fail fast is the right way.

Reply:
Config shouldn't be checked. I think it was for debugging and then forgot to remove...

Reply:
Config shouldn't be checked. I think the check was left in by accident after debugging as it's related to the test fixes (faulty configs).